platform/core/src/flatbuffers_generated/mod.rs

```diff
@@ -0,0 +1,8 @@
+#[allow(unused_imports)]
```

Comment:
Nit: Wouldn't it be relatively straightforward to also generate this file, while at it? Not important though.

Reply:
Yes, it could be, didn't think about this. I will see how it can be done.